orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/aws/cloudformation/WaitForCloudFormationCompletionTask.java

```diff
@@ -67,9 +67,12 @@ public TaskResult execute(@Nonnull Stage stage) {
               + stack.get("stackStatus"));
       boolean isChangeSet =
           (boolean) Optional.ofNullable(stage.getContext().get("isChangeSet")).orElse(false);
+      boolean isChangeSetDeploy =
```

Comment:
What is the meaning of this flag?

Reply:
That flag is set by  [ExecuteCloudFormationChangeSetTask:101](https://github.com/sergiopena/orca/blob/execute_changesets/orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/aws/cloudformation/ExecuteCloudFormationChangeSetTask.java#L101) is used to distinguish when we are creating the ChangeSet from when it's being deployed, so we can reuse the logic and wait for changeSets execution too. Shall I change the name to isChangeSetExecution to make it clearer?

Reply:
Renamed to isChangeSetExecution